### PR TITLE
Expand Apex!

### PR DESCRIPTION
**Pull request by dave-tucker**
This adds two new namespaces:
- apex-qa (for our QA environment)
- apex-monitoring (where we'll be setting necessary prometheus/grafana/jaeger resources to monitor the apex and apex-qa namespaces)

It updates our `apex` ArgoCD application based on a incoming code change, and adds a new `apex-qa` application that handles our staging environment deployment.

### CODE REVIEW

File: cluster-scope/base/core/namespaces/apex-monitoring/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yaml
+- ../../../../components/project-admin-rolebindings/apex
+namespace: apex
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/apex-monitoring/kustomization.yaml:6]:
```suggestion
namespace: apex-monitoring
```

File: cluster-scope/base/core/namespaces/apex-qa/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yaml
+- ../../../../components/project-admin-rolebindings/apex
+namespace: apex
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/apex-qa/kustomization.yaml:6]:
```suggestion
namespace: apex-qa
```

**Review by tumido — CHANGES_REQUESTED** [review]
Nice! :slightly_smiling_face: I have 2 inline suggestions (the `namespace:` value in `kustomization.yaml` has to match the namesapce you're creating, otherwise all the components listed in `components:` would deploy to wrong namespace)

Also please add the new namespaces after this line here, so the resources are properly discovered by ArgoCD and deployed:
https://github.com/operate-first/apps/blob/5cb916a5b16467e061ba2c1f4c39c19554b96a5b/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml#L24

**Comment by dave-tucker** [comment]
Thanks @tumido, changes applied.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**